src/components/views/rooms/WhoIsTypingTile.tsx

```diff
@@ -17,7 +17,6 @@ limitations under the License.
 
 import React from "react";
 import { Room, RoomEvent, RoomMember, RoomMemberEvent, MatrixEvent } from "matrix-js-sdk/src/matrix";
-import { compare } from "matrix-js-sdk/src/utils";
```

Comment:
`matrix-js-sdk/src/utils` is intended to be reusable though not necessarily stable. Given we use a load of utils from that export why are you picking on this one in particular?

![image](https://github.com/user-attachments/assets/274ed793-6bc4-4d0c-badf-c4f73075505c)

Reply:
Since it was instantiated at the module level, it was running before modernizr when I was trying to write the test. I figure rather than rearranging stuff to try to get the execution order right, the best way to fix this was to just not arbitrarily instantiate a static collator and instead make one when we need it, at which point the compare function is adding no value at all.

Reply:
But element-web can't assume any of its dependencies don't do that. Sure we can fix that in the js-sdk but not in other dependencies. So instead we should write our code to defensively handle that situation.

Reply:
I don't disagree, although I'd need to dig more into exactly what order things get run in to do so. This felt like a thing that was still worth doing anyway.

Reply:
Its a case of removing the import here https://github.com/element-hq/element-web/blob/develop/src/vector/url_utils.ts#L17 given it is imported by https://github.com/element-hq/element-web/blob/develop/src/vector/index.ts#L24C40-L24C49

Reply:
Although that said, if you strongly think that's a better thing to do than this then I'll stop killing myself trying to get the test coverage on this PR up to the bar.

Reply:
I think this is a band-aid for a single cause, which is quite esoteric given that https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator#browser_compatibility has been supported for years, so instead making the "bootloader" resilient to any such calls for more recent APIs in dependencies we control and those we do not would be more sane IMO

Reply:
Oh well, test didn't involve fixing other app bugs like the other one did, so this may as well live as tidy-up fix anyway. We might want to consider just loading a small stub from the HTML and loading the full app as an async import, which would give us more flexibility to do loading screens and such as well as have complete control over what executes in the stub.